doc/game_design.md

```diff
@@ -0,0 +1,48 @@
+# Game Design
+Drakkar is a multiplayer game featuring dragons of different colours trying to chase and eat one another.
+The game resembles [agar.io](https://agar.io/) with the addition of a Rock-Paper-Scissor dynamic via the RGB colours.
+
+The goals and rules of the game are described here. We aim to make the design described in this doc come to life, and should refrain from working on things that do not fall in its scope. Any new feature or idea should be discussed first, then if received positively should be included in the design doc. 
+
+## General Principle
+There are three players. Each player controls a disc-shaped dragon of one of three colours: red, green, and blue.
+Every colour is strong against another one and weak against the third one. 
+Players chase one another, trying to run into and eat the weaker player while evading the stronger player.
+
+### Interactions
+A player running into another one triggers a collision. 
+Upon a collision, the colour of the involved dragons is compared, and the dragon with the strongest colour wins, according to the following Rock-Paper-Scissor logic:
```

Comment:
"and the dragon with the strongest colour wins"

Maybe 'stronger' is better here? Because there is actually no strongest color, they are only stronger comparing to the colors that are weaker than them

Reply:
ah, good point!